frontend/src/components/Issue/activity/ActionSentence.vue

```diff
@@ -34,6 +35,26 @@ const { t } = useI18n();
 const renderActionSentence = () => {
   const { activity, issue } = props;
   if (activity.type.startsWith("bb.issue.")) {
+    if (activity.type === "bb.issue.comment.create") {
+      const payload = activity.payload as ActivityIssueCommentCreatePayload;
+      if (payload.externalApprovalEvent) {
+        if (payload.externalApprovalEvent.action == "REJECT") {
+          console.log(payload.externalApprovalEvent);
+          switch (payload.externalApprovalEvent.type) {
+            case "bb.plugin.app.feishu":
+              return t("activity.sentence.external-approval-rejected", {
+                stageName: payload.externalApprovalEvent.stageName,
+                imName: t("common.feishu"),
+              });
+            default:
+              return t("activity.sentence.external-approval-rejected", {
+                stageName: payload.externalApprovalEvent.stageName,
+                imName: "",
+              });
```

Comment:
I guess this part can be refactored a little bit. Like the switch-case part only determine what the `imName` is, and  building the sentence is a common logic outside switch-case here.